internal/service/ipam/api_server_data_source_test.go

```diff
@@ -0,0 +1,131 @@
+		resource.TestCheckResourceAttrPair(resourceName, "gss_tsig_fallback", dataSourceName, "results.0.gss_tsig_fallback"),
+		resource.TestCheckResourceAttrPair(resourceName, "header_option_filename", dataSourceName, "results.0.header_option_filename"),
+		resource.TestCheckResourceAttrPair(resourceName, "header_option_server_address", dataSourceName, "results.0.header_option_server_address"),
+		resource.TestCheckResourceAttrPair(resourceName, "header_option_server_name", dataSourceName, "results.0.header_option_server_name"),
+		resource.TestCheckResourceAttrPair(resourceName, "hostname_rewrite_char", dataSourceName, "results.0.hostname_rewrite_char"),
+		resource.TestCheckResourceAttrPair(resourceName, "hostname_rewrite_enabled", dataSourceName, "results.0.hostname_rewrite_enabled"),
+		resource.TestCheckResourceAttrPair(resourceName, "hostname_rewrite_regex", dataSourceName, "results.0.hostname_rewrite_regex"),
+		resource.TestCheckResourceAttrPair(resourceName, "id", dataSourceName, "results.0.id"),
+		resource.TestCheckResourceAttrPair(resourceName, "inheritance_sources", dataSourceName, "results.0.inheritance_sources"),
+		resource.TestCheckResourceAttrPair(resourceName, "kerberos_kdc", dataSourceName, "results.0.kerberos_kdc"),
+		resource.TestCheckResourceAttrPair(resourceName, "kerberos_keys", dataSourceName, "results.0.kerberos_keys"),
+		resource.TestCheckResourceAttrPair(resourceName, "kerberos_rekey_interval", dataSourceName, "results.0.kerberos_rekey_interval"),
+		resource.TestCheckResourceAttrPair(resourceName, "kerberos_retry_interval", dataSourceName, "results.0.kerberos_retry_interval"),
+		resource.TestCheckResourceAttrPair(resourceName, "kerberos_tkey_lifetime", dataSourceName, "results.0.kerberos_tkey_lifetime"),
+		resource.TestCheckResourceAttrPair(resourceName, "kerberos_tkey_protocol", dataSourceName, "results.0.kerberos_tkey_protocol"),
+		resource.TestCheckResourceAttrPair(resourceName, "name", dataSourceName, "results.0.name"),
+		resource.TestCheckResourceAttrPair(resourceName, "server_principal", dataSourceName, "results.0.server_principal"),
+		resource.TestCheckResourceAttrPair(resourceName, "tags", dataSourceName, "results.0.tags"),
+		resource.TestCheckResourceAttrPair(resourceName, "updated_at", dataSourceName, "results.0.updated_at"),
+		resource.TestCheckResourceAttrPair(resourceName, "vendor_specific_option_option_space", dataSourceName, "results.0.vendor_specific_option_option_space"),
+	}
+}
+
+func testAccServerDataSourceConfigFilters(name string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test" {
+  name = %q
+}
+
+data "bloxone_dhcp_server" "test" {
+  filters = {
+	name = bloxone_dhcp_server.test.name
+  }
+}
+`, name)
+}
+
+func testAccServerDataSourceConfigTagFilters(name string, tagValue string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test" {
+  name = %q
+  tags = {
+	tag1 = %q
+  }
+}
+
+data "bloxone_dhcp_server" "test" {
```

Comment:
```suggestion
data "bloxone_dhcp_servers" "test" {
```

Reply:
Did you run acceptance locally? This would have caused a failure.

Reply:
updated to servers from server